app/src/main/java/de/danoeh/antennapod/fragment/NewEpisodesFragment.java

```diff
@@ -97,7 +97,7 @@ public void onHide(FeedItemUndoToken token) {
                     long itemId = token.getFeedItemId();
                     FeedItem item = DBReader.getFeedItem(context, itemId);
                     FeedMedia media = item.getMedia();
-                    if(media != null && media.hasAlmostEnded() && UserPreferences.isAutoDelete()) {
+                    if(media != null && media.hasAlmostEnded() && item.getFeed().getPreferences().getCurrentAutoDelete(UserPreferences.isAutoDelete())) {
```

Comment:
The reason I haven't done this in the first place is that currently FeedPreferences does not import UserPreferences, while NewEpisodeFragment and all the other files using getCurrrentAutoDelete() already does, so it looked cleaner not to add extra dependency.

The advantage of your suggestion is that it saves adding this parameter in at least 5 places where getCurrrentAutoDelete() is called.
You could say that since this function has no use other than with UserPreferences.isAutoDelete() as a parameter, then maybe it is not worth keeping the parameter.

And now a side git question:
Lets say I fix this,should I squash this commit into my pull request commit, or leave it as another commit on top of the current PR?